content/collections/data/en/autocapture.md

```diff
@@ -54,29 +54,89 @@ Keeping your taxonomy clean and organized is essential to ensuring users can fin
-* The exception to these attribute collection rules is when an element has an explicit attribute added with the prefix “data-amp-track-”. This allows data in these attributes to be intentionally passed back to Amplitude.
+* Autocapture captures the text your website or app displays. For example, the content (`textContent`) of the element clicked and its children. It's not recommended that you use Autocapture's element interaction tracking on pages that may contain sensitive information. Amplitude uses pattern matching to automatically exclude from your collection any text content that looks like a credit card number, social security number, or email address.
+* The exception to these attribute collection rules is when an element has an explicit attribute added with the prefix *data-amp-track-*. This allows data in these attributes to be intentionally passed back to Amplitude.
 * Autocapture automatically removes value, event handlers, style, and react attributes.
 
+Additional autocapture protections include: 
+
+#### Precise text and title masking
+
+Precise text, otherwise known as title masking, excludes text capture from specific elements on the page, including the page title. This lets Autocapture usage on the page without capturing the text displayed in the specific element.
+
+To prevent capturing the text, add the attribute `data-amp-mask` to the element. If, for example you have the following on a button: 
+
+```html
+<div data-amp-mask>John Doe</div>
+```
+The above example still tracks the click on the div. However, the text content "John Doe" doesn't appear in the Autocapture data. This exclusion works on the SDK level. 
+
+Precise text works recursively so that any text contained in an element is masked. For example: 
+
+```html
+<div data-amp-mask>
+  John Doe
+  <div>Jane Doe</div>
+</div>
+```
+In the above example, both names are masked with `*****`.
+
+#### Email address masking
+
+Email address masking automatically hides or otherwise obscures email addresses. Amplitude uses pattern matching to automatically exclude content that looks like an email address. Email address masking is recursive. Any email addresses captured within an element is masked such as `*****`.
+
+#### Pattern (Regex) masking
+
+You can mask information based on patterns that you specify (regular expressions or Regex). If you aren't familiar with Regex, review this page on [regular expressions](https://www.regular-expressions.info/quickstart.html).
+
+You can specify a pattern of information that you want to mask within Amplitude. This configuration occurs in the SDK and is an additional layer of protection to the default patterns Amplitude uses to mask email, credit cards, and social security numbers. Regex filters mask values in any fields where it may be possible to include this data. This includes both visible fields as well as hidden attributes on the page. 
```

Comment:
```suggestion
Specify a pattern of information that you want Amplitude to mask. This configuration happens in the SDK and is an additional layer of protection to the default patterns Amplitude uses to mask email, credit cards, and social security numbers. Regex filters mask values in any fields where it may be possible to include this data. This includes both visible fields as well as hidden attributes on the page. 
```
A link to the SDK doc sections wherever you reference the SDK will be helpful.